pkg/crd/spec.go

```diff
@@ -138,18 +138,23 @@ func (p *Parser) NeedCRDFor(groupKind schema.GroupKind, maxDescLen *int) {
 		crd.Spec.Versions[0].Storage = true
 	}
 
-	hasStorage := false
+	storageCounter := 0
+
 	for _, ver := range crd.Spec.Versions {
 		if ver.Storage {
-			hasStorage = true
-			break
+			storageCounter++
 		}
```

Comment:
Suggestion for you does need to continue the for unnecessary.  ( replace 141 - 157 for )

```go
	hasStorage := false
	for _, ver := range crd.Spec.Versions {
		if ver.Storage {
			if hasStorage {
				// just add the error to the first relevant package for this CRD,
				// since there's no specific error location
				packages[0].AddError(fmt.Errorf("CRD for %s has more than one storage version", groupKind))
			}
			hasStorage = true
			break
		}
	}
	if !hasStorage {
		// just add the error to the first relevant package for this CRD,
		// since there's no specific error location
		packages[0].AddError(fmt.Errorf("CRD for %s has no storage version", groupKind))
	}

```